### ISSUE TITLE
Add images to your Article Schema throws fatal error 

### ISSUE THREAD

**Issue by shabnam611** [Yoast/developer#308]
### Please give us a description of the issue
When I use the filter wpseo_schema_article and use the example code provided [here](https://developer.yoast.com/features/schema/pieces/article/#add-images-to-your-article-schema) I see fatal error. 

Steps to replicate: 

1. I used to code snippet plugin to add the example code 
2. Changed the value of $attachment_id in the example. I added an image and used the ID for that image
3. Then I saved the code and activated it. 
4. I checked a post where I hadset schema type as webpage and page type as article. I noticed fatal error in the page source

Screenshot: 
<img width="1165" alt="Screenshot 2024-06-22 at 4 02 34 PM" src="https://github.com/Yoast/developer/assets/5788402/610f1305-5f87-4b48-9806-3c66657d42cf">

* [ ] I've read and understood the [contribution guidelines](https://github.com/Yoast/developer/blob/main/.github/CONTRIBUTING.md).
* [ ] I've searched for any related issues and avoided creating a duplicate issue.

**Comment by mhkuu** [comment]
Good call! Yeah, that should probably use the `generate_from_attachment_id` function instead, as referenced below the code snippet. I will test that and update the snippet later.
